src/KustoExpressionParser.ts

```diff
@@ -46,9 +46,11 @@ export class KustoExpressionParser {
     this.appendProperty(context, query.expression.from, parts);
     this.appendTimeFilter(context, undefined, parts, tableSchema);
 
-    const where = replaceByIndex(query.index, query.expression.where, query.search);
+    if (query.index) {
+      const where = replaceByIndex(query.index, query.expression.where, query.search);
```

Comment:
This `replaceByIndex` is meant to get all the `where` clauses from a query, and remove the one that is being edited at the moment, to get a valid query to get suggestions.

The issue is that the new `EditorList` component does not provide the index of each element so I cannot make this replacement. What this means is that, for suggestions, we are not taking current filters to filter out suggestions.